src/main/java/com/datatheorem/mobileappsecurity/jenkins/plugin/SendBuildToDataTheoremPublisher.java

```diff
@@ -66,6 +69,8 @@ public SendBuildToDataTheoremPublisher(
         this.proxyPort = proxyPort;
         this.proxyUsername = proxyUsername;
         this.proxyPassword = Secret.fromString(proxyPassword);
+        this.applicationCredentialUsername = applicationCredentialUsername;
```

Comment:
A third field is supported, but not included here (`comments`). Also, these fields could also be grouped.